packages/plugin-ext/src/main/browser/webview/pre/main.js

```diff
@@ -283,6 +285,13 @@
             });
         };
 
+        function preventDefaultBrowserHotkeys(e) {
+            // F1 or CtrlCmd+P
+            if (e.keyCode === 112 || ((e.ctrlKey || e.metaKey) && e.keyCode === 80)) {
```

Comment:
`e.ctrlKey || e.metaKey` -> `(e.ctrlKey && !isOSX) || (e.metaKey && isOSX)`?

Reply:
We cannot use `isOSX` here 🙈 there should be a better solution without reimplementing our keybinding support in the webview iframe.

Reply:
Out of curiosity, cannot you ask for `navigator.userAgent` or `process.platform`? One of them should be available.

Reply:
yes we can replace with `navigator.userAgent`

Reply:
Used `navigator.platform` to determine user operation system, it is more confident.